DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/README.md

```diff
@@ -424,6 +427,32 @@ Summarizing bandwidth in MB/s/bank for banks 1 to 8
 
 KERNEL-TO-MEMORY BANDWIDTH = 16509.6 MB/s/bank
 
+*****************************************************************
+***********************  USM Bandwidth  *************************
+*****************************************************************
+
+Case: Full Duplex
+Iterations: 1
+Data size: 1024 MB
+Data type size: 64 bytes
+Average Time: 99122.8 ns	
+Average Throughput: 20.177 GB/s	
+
+Case: From Host to Device
+Iterations: 1
+Data size: 1024 MB
+Data type size: 64 bytes
+Average Time: 86711.5 ns	
+Average Throughput: 11.5325 GB/s	
+
+Case: From Device to Host
+Iterations: 1
+Data size: 1024 MB
+Data type size: 64 bytes
+Average Time: 84240.3 ns	
+Average Throughput: 11.8708 GB/s	
+
+
 BOARD TEST PASSED

```

Comment:
does it make sense to summarize the board test data in a table? If someone is using this to benchmark their custom BSP, i could see them needing to present that type of info to their manager so if we present a nice tab-separated table or a markdown-formatted table, they can copy-paste it easily. feel free to disregard this suggestion :)

Reply:
possibly! we can consider adding this as part of the second PR for all the tests